common/extend-visa.md

```diff
@@ -0,0 +1,39 @@
+# What I need to extend my Visa?
+
+## Rot-Weiß-Rot Karte (Red-White-Red Card)
+**TBD**
+
+## Rot-Weiß-Rot plus Karte (Red-White-Red plus Card)
+To initiate the process of extending the RWR plus card you need to proceed to the [MA 35](https://www.wien.gv.at/kontakte/ma35/) office that covers the district your are currently living. It is recommended that you start this process at least 3 months before your currently Visa date expiration.
```

Comment:
typos: 
- "**you are** currently living **in**"
- "your current visa expires" or "your current visa expiration date"